packages/eas-cli/src/credentials/android/utils/printCredentials.ts

```diff
@@ -44,10 +44,31 @@ function displayAndroidFcmCredentials(appCredentials: CommonAndroidAppCredential
   Log.newLine();
 }
 
+function displayGoogleServiceAccountKeyForSubmissions(
+  appCredentials: CommonAndroidAppCredentialsFragment
+): void {
+  const maybeGsaKey = appCredentials.googleServiceAccountKeyForSubmissions;
+  Log.log(`  Google Service Account Key For Submissions:`);
+  if (!maybeGsaKey) {
+    Log.log(`    None assigned yet`);
+    Log.newLine();
+    return;
+  }
+  const { projectIdentifier, privateKeyIdentifier, clientEmail, clientIdentifier, updatedAt } =
+    maybeGsaKey;
+  Log.log(`    Project Id: ${projectIdentifier}`);
+  Log.log(`    Private Key Id: ${privateKeyIdentifier}`);
+  Log.log(`    Client Email: ${clientEmail}`);
+  Log.log(`    Client Id: ${clientIdentifier}`);
+  Log.log(`    Updated ${fromNow(new Date(updatedAt))} ago`);
```

Comment:
We have a [`formatFields()`](https://github.com/expo/eas-cli/blob/cf11ae93af07ffde075a7bc80680bf5f5338108e/packages/eas-cli/src/utils/formatFields.ts#L7) method for this.
Example usages:
- [Submissions summary](https://github.com/expo/eas-cli/blob/cf11ae93af07ffde075a7bc80680bf5f5338108e/packages/eas-cli/src/submit/utils/summary.ts#L64) - uses custom format options
- [Device list](https://github.com/expo/eas-cli/blob/cf11ae93af07ffde075a7bc80680bf5f5338108e/packages/eas-cli/src/devices/utils/formatDevice.ts#L8) - almost copy-paste example for this case 😉

Reply:
wow it looks so much better now 😮